inst/rmarkdown/templates/ajs_outbreak/skeleton/skeleton.Rmd

```diff
@@ -1581,126 +1890,160 @@ mortality_rate(deaths$deaths, deaths$population, multiplier = 10000) %>%
 
 
 #### Maps 
+<!-- ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+read_shapefiles
 
+To create maps, you need to have a shapefile of the area. Often, the MSF GIS
+unit can provide shapefiles.
 
+Your shapefile can be a polygon or points. Polygons do not need to be contiguous.
 
-```{r read_shapefiles, message=FALSE}
-
-# DELETE THIS LINE IF YOU HAVE YOUR OWN SHAPEFILE
-# generate fake shapefile
-map <- gen_polygon(regions = unique(linelist_cleaned$patient_origin))
```

Comment:
removing this causes the mapping section to fail

Reply:
maybe move it up to the section where fake data is generated?

Reply:
yep, having it in the fake data chunk doesn't work, as linelist_cleaned is not yet loaded. moved back to the map section.